official/vision/beta/serving/run_lib.py

```diff
@@ -133,3 +138,97 @@ def get_export_module(experiment: str,
         type(params.task)))
 
   return export_module
+
+
+def read_class_names(class_names_path):
+  """Reads class names from text file.
+  Supports .txt and .json.
+  
+  Args:
+    class_names_path: `str`, path to json/txt file containing classes. 
+      Text file should contain one class name per line.
+      Json file should contain only one dictionary, `Mapping[int, str]`
+  """
+  
+  names = {}
+  if class_names_path.endswith('.txt'):
+    with open(class_names_path, 'r') as data:
+      for idx, name in enumerate(data):
+        names[idx] = name.strip('\n')
+  
+  elif class_names_path.endswith('.json'):
+    with open(class_names_path) as f:
+      names = json.load(f)
+    if type(list(names.keys())[0]) == str and type(list(names.values())[0]) == int:
+      names = dict((v,k) for k,v in names.items())
+
+  else:
+    raise NotImplementedError('File type is not .txt or .json, path %s' %class_names_path)
+
+  assert type(list(names.keys())[0]) == int, 'Loaded dict %s has wrong key type %s' %(
+    class_names_path, type(list(names.keys()[0])))
+  assert type(list(names.values())[0]) == str, 'Loaded dict %s has wrong value type %s' %(
+    class_names_path, type(list(names.values()[0])))
+
+  return names
+
+
+def draw_bbox(image: np.array, 
+              bboxes: np.array,
+              scores: np.array,
+              classes: np.array,
+              num_bboxes: np.array,
+              class_names: Mapping[int, str], 
+              show_label: bool = True):
+  num_classes = len(class_names)
+  image_h, image_w, _ = image.shape
+  hsv_tuples = [(1.0 * x / num_classes, 1., 1.) for x in range(num_classes)]
+  colors = list(map(lambda x: colorsys.hsv_to_rgb(*x), hsv_tuples))
+  colors = list(map(lambda x: (int(x[0] * 255), int(x[1] * 255), int(x[2] * 255)), colors))
+
+  random.seed(0)
```

Comment:
just curious why do we need to shuffle?

Reply:
To fix a color palette according to seed. Forgot to make the seed an argument. Did so and documented in changes below.